lms/djangoapps/courseware/courses.py

```diff
@@ -413,6 +413,13 @@ def sort_by_start_date(courses):
     return courses
 
 
+def sort_by_course_id(courses):
+    """
+    Returns a list of courses sorted by their course_id.
+    """
+    return sorted(courses, key=lambda x: str(x.pk).lower(), reverse=False)
```

Comment:
reverse=False is the default value, could be omitted.

Reply:
I deleted this value.